tools/istio-docker.mk

```diff
@@ -230,7 +230,12 @@ docker.node-agent-test: $(ISTIO_DOCKER)/node_agent.key
 # DOCKER_BUILD_VARIANTS:=default distroless
 DOCKER_BUILD_VARIANTS:=default
 DEFAULT_DISTRIBUTION=default
-DOCKER_RULE=$(foreach VARIANT,$(DOCKER_BUILD_VARIANTS), time (mkdir -p $(DOCKER_BUILD_TOP)/$@ && cp -r $^ $(DOCKER_BUILD_TOP)/$@ && cd $(DOCKER_BUILD_TOP)/$@ && $(BUILD_PRE) docker build $(BUILD_ARGS) --build-arg BASE_DISTRIBUTION=$(VARIANT) -t $(HUB)/$(subst docker.,,$@):$(subst -$(DEFAULT_DISTRIBUTION),,$(TAG)-$(VARIANT)) -f Dockerfile$(suffix $@) . ); )
+DOCKER_RULE=$(foreach VARIANT,$(DOCKER_BUILD_VARIANTS), \
+                time (mkdir -p $(DOCKER_BUILD_TOP)/$@ && \
```

Comment:
this change has a whole bunch of errant white space.  can you revert or simplify?

Reply:
Since I'm rather new to make I'm not sure what steps could be taken to simplify the code (e.g. are there make intern variables?). Do you have any suggestions?
Reverting the change is of course still an option, but in IMHO it would make readability worse again and we have a similar problem [some lines beneath](https://github.com/istio/istio/pull/14737/files/49f039fcbdd3d46bef8aa3415d2f2a5da29b2705#r293796602).

Reply:
This change was reverted. resolving conversation.